packages/lesswrong/lib/rateLimits/utils.ts

```diff
@@ -156,13 +126,12 @@ export function calculateRecentKarmaInfo(userId: string, allVotes: RecentVoteInf
 
 function getRateLimitName (rateLimit: AutoRateLimit) {
   let rateLimitName = `${rateLimit.itemsPerTimeframe} ${rateLimit.actionType} per ${rateLimit.timeframeLength} ${rateLimit.timeframeUnit}`
-  const thresholdInfo = rateLimitThresholds.map(threshold => rateLimit[threshold] ? `${rateLimit[threshold]} ${threshold.replace("Threshold", "")}` : undefined).filter(threshold => threshold)
-  return rateLimitName += ` (${thresholdInfo.join(", ")})`
+  return rateLimitName += ` (via function: ${rateLimit.isActive.toString()})`
 }
 
 function getActiveRateLimits<T extends AutoRateLimit>(user: UserKarmaInfo & { recentKarmaInfo: RecentKarmaInfo }, autoRateLimits: T[]) {
   const nonUniversalLimits = autoRateLimits.filter(rateLimit => rateLimit.rateLimitType !== "universal")
-  return nonUniversalLimits.filter(rateLimit => shouldRateLimitApply(user, rateLimit, user.recentKarmaInfo))
+  return nonUniversalLimits.filter(rateLimit => (!rateLimit.isActive(user, user.recentKarmaInfo)))
```

Comment:
Is this passing a type check, given that `user.recentKarmaInfo` is of type `RecentKarmaInfo`, which is missing `downvoteRatio` (as would be required for `RateLimitFeatures`?